docs/advanced-topics/advanced-features.rst

```diff
@@ -9,7 +9,7 @@ provisioning for inputs and outputs.
 
 File provisioning for some protocols, like HTTP and FTP, is built-in
 while other protocols are handled by plugins as documented
-`here <https://github.com/ga4gh/dockstore/tree/develop/dockstore-file-plugin-parent>`__.
```

Comment:
for `docs/_static/2017_BOSC_Sanger_keiranmraine.pdf` why is the binary size changing?

Reply:
Hmmm, looking at the file. I think the file was maybe corrupted when copying from one location to another? 
https://raw.githubusercontent.com/dockstore/dockstore-documentation/819bf7f80d2d37ef365c8715f506d1ead98dcf23/docs/_static/2017_BOSC_Sanger_keiranmraine.pdf

Reply:
Weird, I'll fix that